packages/esm-ward-app/src/hooks/useInstalledModules.ts

```diff
@@ -0,0 +1,12 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
```

Comment:
Can we add this to esm-api instead?  Why put this in patient-management?

Reply:
esm-react-utils but yes, thanks!

Reply:
I think https://github.com/openmrs/openmrs-esm-core/pull/1028 covers the use-case for this.